mods/tuxemon/maps/spyder_leather_town.tmx

```diff
@@ -396,13 +396,19 @@
     <property name="cond2" value="is player_facing up"/>
    </properties>
   </object>
-  <object id="301" name="Environment" type="event" x="16" y="0" width="16" height="16">
+  <object id="301" name="Environment Day" type="event" x="0" y="0" width="16" height="16">
```

Comment:
Doesn't multiple objects on the same square make it difficult to read?

Reply:
I think you're right, I'll update it.